src/api/content.js

```diff
@@ -0,0 +1,18 @@
+import Tabletop from 'tabletop';
+import log from '../utils/log';
+
+export function getFields() {
+  return (
+    new Promise((resolve) => {
+      Tabletop.init({
+        key: '18yAMk_ydGpPLHTZPrLox7oplvgc-4aswu1arO_IHY9I',
```

Comment:
Could use a comment explaining what this key corresponds to. Presumably it is a google sheet, but it doesn't look like anything in this file explains that at all